src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/NewAzureRmRecoveryServicesAsrReplicationProtectedItem.cs

```diff
@@ -508,7 +560,14 @@ private void VMwareToAzureReplication(EnableProtectionInput input)
                 EnableRdpOnTargetOption = Constants.NeverEnableRDPOnTargetOption,
                 DiskEncryptionSetId = this.DiskEncryptionSetId,
                 TargetAvailabilityZone = this.RecoveryAvailabilityZone,
-                TargetProximityPlacementGroupId = this.RecoveryProximityPlacementGroupId
+                TargetProximityPlacementGroupId = this.RecoveryProximityPlacementGroupId,
+                TargetAvailabilitySetId = this.RecoveryAvailabilitySetId,
+                TargetVmSize = this.Size,
+                SqlServerLicenseType = this.SqlServerLicenseType,
+                TargetVmTags = this.RecoveryVmTag,
+                TargetNicTags = this.RecoveryNicTag,
+                SeedManagedDiskTags = this.DiskTag,
```

Comment:
>SeedManagedDiskTags = this.DiskTag [](start = 16, length = 34)

If RecoveryAzureStorageAccountId is not null or string.empty, then also we shd set this. Otherwise service will throw exception.
@anmolbhatia289

Reply:
Yes, that is correct

---
In reply to: [601546593](https://github.com/AsrOneSdk/azure-powershell/pull/21#discussion_r601546593) [](ancestors = 601546593)

Reply:
Handled in powershell itself.

---
In reply to: [601546593](https://github.com/AsrOneSdk/azure-powershell/pull/21#discussion_r601546593) [](ancestors = 601546593)